CHANGELOG.md

```diff
@@ -2,6 +2,81 @@
 
 Notable changes to Apisonator will be tracked in this document.
 
+## 2.86.0 - 2018-04-25
+
+### Added
+
+- The metrics used for reporting transactions and authorizations to the master
+  account are now configurable under `master.metrics.transactions` and
+  `master.metrics.transactions_authorize`.
+- There is now a set of rake targets for selectively run tests and specs by type
+  (ie. integration, unit, etc) and by specific file.
+- The `make dev` command supports specifying an env variable `PORT` to expose
+  the listener in that port on the local machine. See README.md for details.
+- Miscellaneous improvements in test coverage of previously undertested code.
+
+### Changed
+
+- Several error conditions that produced empty responses now behave like other
+  errors and output valid XML.
+- Added extra measures to avoid creating applications through the internal API
+  without mandatory attributes.
+
+### Fixed
+
+- Running make in systems with a non-GNU version of `time` should stop failing
+  because of using unsupported options.
+- Fixed a bug that made extra requests for stats with the inexistent `seconds`
+  period.
+- Fixed a bug in dealing with the `notification_batch` configuration that made
+  Apisonator enqueue NotifyJobs continuously when the setting was left
+  unspecified.
+
+### Shortlog v2.85.0..v2.86.0
```

Comment:
I'm not sure it's worth it to have all the commits here. Maybe it would be more useful to have links to PRs in the section above.

Reply:
I am not very convinced of this either, and linking to PRs is a good idea (and we would need to change the text in the README then), just a bit more work.

This in particular is inspired on similar practices in other projects such as GNU's, which are even more detailed with changes per file. There's another option I think might make sense to consider as well, which is creating a directory for release notes and then adding this commit summary and other information there (ie. back compat stuff, known issues, etc) like the git project does.

It might be too heavy for frequent releases though. Will wait for more feedback from the team.

Reply:
I would attach PR links to previous sections points something like 
```
* Added extra measures to avoid creating applications through the internal API without mandatory attributes. PR #17
```